fog/view/enclave/api/src/lib.rs

```diff
@@ -162,6 +165,13 @@ pub trait ViewEnclaveApi: ReportableEnclave {
         untrusted_query_response: UntrustedQueryResponse,
     ) -> Result<Vec<u8>>;
 
+    /// Service a query response from a frontend.
+    fn query_backend(
```

Comment:
:wrench: should the comment say `backend`?
```suggestion
    /// Service a query response from a backend.
    fn query_backend(
```

Reply:
No, the response is coming from a frontend.